### PR TITLE
solution

### PR DESCRIPTION
**Pull request by DimaMayak**

### CODE REVIEW

File: src/Solution.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Solution {
+    public boolean isSameTree(TreeNode p, TreeNode q) {
+
+        if (p == null && q == null) return true;
```
Comment (Maiakov) [inline src/Solution.java:4]: use better names for input parameters ...for example - left, right, or 
firstTree, secondTree

File: src/TreeNode.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+public class TreeNode {
+    int val;
+    TreeNode left;
+    TreeNode right;
+
+    TreeNode() {
```
Comment (Maiakov) [inline src/TreeNode.java:6]: do we really need empty constructor?

Comment (DimaMayak) [reply src/TreeNode.java:6]: this part of the code is presented by an example on leetcode

File: src/Solution.java (hunk 1 above)
Comment (DimaMayak) [reply src/Solution.java:4]: this part of the code is presented by an example on leetcode

**Pull request merged**